ui/jetbrains/build.gradle

```diff
@@ -17,6 +17,8 @@ dependencies {
         exclude group: "org.slf4j", module: "jcl-over-slf4j"
     }
 
+    compile("org.codehaus.jackson:jackson-mapper-asl:$versionJacksonMapper")
```

Comment:
Old Jackson is used here.
Check out this page: https://github.com/FasterXML/jackson-databind

New jackson is under "com.fasterxml.jackson" package.